src/app/clusters/sample-mei-server/sample-mei-server.h

```diff
@@ -0,0 +1,85 @@
+#ifndef SRC_APP_CLUSTERS_SAMPLE_MEI_CLUSTER_SERVER_SERVER_H_
+#define SRC_APP_CLUSTERS_SAMPLE_MEI_CLUSTER_SERVER_SERVER_H_
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <app/CommandHandlerInterface.h>
+#include <app/ConcreteCommandPath.h>
+#include <app/util/af-types.h>
+#include <app/util/basic-types.h>
+#include <lib/support/Span.h>
+#include <platform/CHIPDeviceConfig.h>
+
+#ifdef ZCL_USING_SAMPLE_MEI_CLUSTER_SERVER
+#define SAMPLE_MEI_NUM_SUPPORTED_ENDPOINTS                                                                                         \
+    (EMBER_AF_SAMPLE_MEI_CLUSTER_SERVER_ENDPOINT_COUNT + CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT)
```

Comment:
Shouldn't this include the header that would define EMBER_AF_SAMPLE_MEI_CLUSTER_SERVER_ENDPOINT_COUNT?  Concretely, `app/util/config.h`.

Reply:
Yes, explicitly including in sample-mei-server.h